python/ray/experimental/channel/shared_memory_channel.py

```diff
@@ -474,12 +475,70 @@ def close(self) -> None:
         Close this channel by setting the error bit on both the writer_ref and the
         reader_ref.
         """
+        print("SANG-TODO close!")
         self._worker.core_worker.experimental_channel_set_error(self._writer_ref)
         if self.is_local_node(self._reader_node_id):
             self.ensure_registered_as_reader()
         self._worker.core_worker.experimental_channel_set_error(self._reader_ref)
 
 
+class _CloseSignal:
+    pass
+
+
+# @DeveloperAPI
+class BufferedRemoteChannel(ChannelInterface):
```

Comment:
Maybe this is just a draft PR, but wondering why you call it a remote channel

Reply:
intra process channel doesn't have to use this mechanism as it is immediately read/written